docs/install-kubefed-operator.md

```diff
@@ -0,0 +1,24 @@
+# Installing the KubeFed Operator
+
+## Installing from the OperatorHub using the web console
+
+This procedure explains how to install and subscribe to the KubeFed Operator from the OperatorHub using the OpenShift Container Platform web console.
+
+### Prerequisites
+- Access to an OpenShift Container Platform cluster using an account with `cluster-admin` permissions.
+- Any namespaces required for a namespace scoped installation must be created before installing the Operator.
+
+### Procedure
+
+1. Login to the OpenShift Container Platform web console and access the **Administrator** console using the dropdown menu.
+1.  Navigate to the **Operators** menu and select **OperatorHub**.
+1. Type **KubeFed** into the filter box to find the KubeFed Operator.
+1. Select the KubeFed Operator to display information about the Operator, then use the **Install** button to install it.
+1. Select an **Installation mode**.
+
+  The installation can be either cluster scoped (the Operator is available on all namespaces in a cluster), or namespace scoped (the Operator is available only on the specified namespace).
```

Comment:
`the Operator will reconcile the KubeFedWebHook and KubeFed CR instances in all namespaces`